__tests__/news-server.test.js

```diff
@@ -96,3 +96,35 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: responds with an array of articles sorted by the newest first", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
```

Comment:
Need to check these are sorted in the specified order - can use jest sorted for this